### ISSUE TITLE
PowerShell Install-Package on Ubuntu 20.04 - Unable to resolve package reference ''.

### ISSUE THREAD

**Issue by JasonJ387** [PowerShell/PowerShell#14853]
I'm using PowerShell with .net core 5.0 in Ubuntu 20.4; but I'm having an issue importing nuget libraries using the Install-Package command.

The machine is in a locked down environment with no access to the outside world so nuget pacakages are stored on a fileshare.

This works fine on Windows Devices, but the Ubuntu device is complaining.

The specific command I'm using is:
Install-Package -Source /mnt/share/DotNetCorePackages -Name system.xml.xpath.xmldocument -SkipDependencies

The error message I'm receiving is:
Install-Package: Unable to resolve package reference ''.

**Comment by daxian-dbw** [comment]
@JasonJ387 This is an issue specific to the `Install-Package` cmdlet, which is from the `PackageManagement` module. Please open the issue in the https://github.com/OneGet/oneget repo instead. Thanks!

**Comment by JiayuanL** [comment]
@JasonJ387  Can I check how did you resolve this issue finally? I ran into this issue now. Thanks!

**Comment by JasonJ387** [comment]
> @JasonJ387 Can I check how did you resolve this issue finally? I ran into this issue now. Thanks!

It’s been a while, and I no longer have access to the code to check.

 I did resolve the issue and get this working but I think the error was a red herring and the problem was with non-compatible dependency calls within the module.  I.e referencing windows specific commands.

i believe I placed an OS check in the script and added branching logic.

hope this helps.
J
